### PR TITLE
Remove remaining usages of gGameCommandErrorText

### PR DESCRIPTION
**Pull request by Gymnasiast**
And gGameCommandErrorTitle.

### CODE REVIEW

File: src/openrct2-ui/windows/RideConstruction.cpp [hunk 1]
```diff
@@ -3469,15 +3470,26 @@ void ride_construction_tooldown_construct(const ScreenCoordsXY& screenCoords)
 
             gDisableErrorWindowSound = true;
 
-            _trackPlaceCost = maze_set_track(
-                CoordsXYZD{ _currentTrackBegin, 0 }, GAME_COMMAND_FLAG_APPLY, true, _currentRideIndex, GC_SET_MAZE_TRACK_BUILD);
+            auto gameAction = MazeSetTrackAction(
+                CoordsXYZD{ _currentTrackBegin, 0 }, true, _currentRideIndex, GC_SET_MAZE_TRACK_BUILD, GAME_COMMAND_FLAG_APPLY);
```
Comment (duncanspumpkin) [inline src/openrct2-ui/windows/RideConstruction.cpp:3474]: Apply flag doesn't mean anything anymore for game actions.

File: src/openrct2/actions/MazeSetTrackAction.cpp [hunk 2]
```diff
@@ -24,12 +24,14 @@
...
 
-MazeSetTrackAction::MazeSetTrackAction(const CoordsXYZD& location, bool initialPlacement, RideId rideIndex, uint8_t mode)
+MazeSetTrackAction::MazeSetTrackAction(
+    const CoordsXYZD& location, bool initialPlacement, RideId rideIndex, uint8_t mode, uint32_t flags)
     : _loc(location)
     , _initialPlacement(initialPlacement)
     , _rideIndex(rideIndex)
     , _mode(mode)
 {
+    SetFlags(flags);
```
Comment (duncanspumpkin) [inline src/openrct2/actions/MazeSetTrackAction.cpp:34]: No other game action sets the flags like this. I'd prefer consistency.

File: src/openrct2-ui/windows/RideConstruction.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2-ui/windows/RideConstruction.cpp:3474]: Can it just be removed everywhere in our code?

Comment (duncanspumpkin) [reply src/openrct2-ui/windows/RideConstruction.cpp:3474]: There are a few places in the codebase where it's used when not calling the game action directly to split on a query or execute.

**Review by IntelOrca — APPROVED** [review]

**Pull request merged**